modules/vaos/spec/services/jwt_spec.rb

```diff
@@ -5,14 +5,58 @@
 describe VAOS::JWT do
   subject { VAOS::JWT.new(user) }
 
-  let(:user) { build(:user, :mhv) }
+  let(:user) { build(:user, :vaos) }
+  let(:rsa_private) { OpenSSL::PKey::RSA.generate(4096) }
+  let(:jwt_regex) { %r{^[A-Za-z0-9\-_=]+\.[A-Za-z0-9\-_=]+\.?[A-Za-z0-9\-_.+/=]*$} }
```

Comment:
😱 

Could you provide a human-readable comment for what this is representing?

Reply:
I think "jwt_regex" is pretty self explanatory. JWTs have 3 parts: a header, a payload, and a signature. Each part is base64url encoded. You want me to put that in the header? I think it's common knowledge for anyone working with JWT, but happy to do it if you think it is helpful.